### PR TITLE
Added Code of Conduct per club request

### PR DESCRIPTION
**Pull request by KingZethor**
Necessary for a proper project to succeed without any scandals.

### CODE REVIEW

**Review by Zachdogg1 — DISMISSED** [review]
So I can edit comments?

**Comment by KingZethor** [comment]
bump

**Review by DLGesus — APPROVED** [review]
Definitely necessary to our project

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```
Comment (Zachdogg1) [inline CODE_OF_CONDUCT.md:5]: no

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at deep.desai@student.allenisd.org. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4, available at [http://contributor-covenant.org/version/1/4][version]
+
+[homepage]: http://contributor-covenant.org
+[version]: http://contributor-covenant.org/version/1/4/
```
Comment (Zachdogg1) [inline CODE_OF_CONDUCT.md:48]: ???

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ErikTheCleric) [reply CODE_OF_CONDUCT.md:5]: Why?

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Watching meme videos instead of working
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
```
Comment (Zachdogg1) [inline CODE_OF_CONDUCT.md:23]: <><><?><>

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (DLGesus) [reply CODE_OF_CONDUCT.md:5]: yes

**Review by Zachdogg1 — COMMENTED** [review]
k

**Pull request merged**